storage-proofs/src/stacked/params.rs

```diff
@@ -362,9 +364,9 @@ impl<H: Hasher, G: Hasher> TemporaryAux<H, G> {
         self.labels.column(column_index)
     }
 
-    // "Compact" will discard all persisted data that is no longer
-    // required and compact the remaining "tree_r_last" merkle tree.
-    pub fn compact(t_aux: TemporaryAux<H, G>) -> Result<()> {
+    // 'delete' will discard all persisted merkle and layer data that
+    // is no longer required.
+    pub fn delete(t_aux: TemporaryAux<H, G>) -> Result<()> {
```

Comment:
We still keep part of the cache, so I think calling it `compact` is still the right name

Reply:
We do, but that method only deletes now. Compaction is done elsewhere, immediately after the trees are built.

Reply:
:+1: Renamed to `clear_temp`